packages/loaders/src/elements/Progress.spec.tsx

```diff
@@ -6,9 +6,14 @@
  */
 
 import React from 'react';
-import { render } from 'garden-test-utils';
+import { render, renderDark } from 'garden-test-utils';
 import { Progress } from './Progress';
-import { PALETTE_V8 } from '@zendeskgarden/react-theming';
+import { DEFAULT_THEME, PALETTE } from '@zendeskgarden/react-theming';
+import { rgba } from 'polished';
+
+function getRenderFn(mode: 'dark' | 'light') {
+  return mode === 'dark' ? renderDark : render;
+}
```

Comment:
Would it be useful to move this into `garden-test-utils`? I reduces verbosity when using `it.each`:

```js
it.each<['light' | 'dark', string]>([
      ['light', PALETTE.grey[700]],
      ['dark', PALETTE.grey[500]]
    ])('applies the default colors in "%s mode', (mode, color) => {
      const { container } = getRenderFn(mode)(<Component ... />);
      ...
})
```

Reply:
I'm ok with adding to the test utils.

Reply:
To avoid blowing scope, I'll do that refactor in a small separate PR since I'm also using this pattern in [Tooltip](https://github.com/zendeskgarden/react-components/pull/1811/files#diff-9ca09c7b035971f09ea9d6cf8f106c45fdfa61041e2986f36ae778e3dc78731aR15-R18).